<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="13"
           failures="5"
           errors="0"
           skipped="0"
           time="789.328">
  <properties />
            <testcase name="A_out5_fxo" time="33.792" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound">
                <failure type="java.lang.AssertionError"
                  message="CDR源中继检测 【预期：  实际：FXO2-3】"
              >
<![CDATA[java.lang.AssertionError: CDR源中继检测 【预期：  实际：FXO2-3】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:254)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:242)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound.A_out5_fxo(Outbound.java:89)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="A_out6_bri" time="33.729" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound">
                <failure type="java.lang.AssertionError"
                  message="CDR源中继检测 【预期：  实际：BRI2-1】"
              >
<![CDATA[java.lang.AssertionError: CDR源中继检测 【预期：  实际：BRI2-1】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:254)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:242)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound.A_out6_bri(Outbound.java:100)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E_timecondition" time="65.613" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound">
                <failure type="java.lang.AssertionError"
                  message="CDR被叫方检测 【预期：9299999 实际：933333】"
              >
<![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：9299999 实际：933333】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:251)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:242)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound.E_timecondition(Outbound.java:248)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="A_out7_E1" time="34.367" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound">
                <failure type="java.lang.AssertionError"
                  message="CDR源中继检测 【预期：  实际：DIGIT1】"
              >
<![CDATA[java.lang.AssertionError: CDR源中继检测 【预期：  实际：DIGIT1】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:254)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:242)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound.A_out7_E1(Outbound.java:111)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="A_out8_gsm" time="41.081" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound">
                <failure type="java.lang.AssertionError"
                  message="CDR源中继检测 【预期：  实际：UMTS3-3】"
              >
<![CDATA[java.lang.AssertionError: CDR源中继检测 【预期：  实际：UMTS3-3】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1086)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:579)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:142)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:116)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:199)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:254)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:242)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound.A_out8_gsm(Outbound.java:122)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="F_delete" time="87.877" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="A_out3_sps" time="38.490" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="B_Prepend" time="63.833" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="A_out2_iax" time="41.100" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="D_rrmemory_hunt" time="163.392" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="A_out4_sps" time="41.186" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="C_password" time="108.659" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
                <testcase name="A_out1_sip" time="36.209" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcore.Outbound" />
      
</testsuite>
